autolux/opts.py

```diff
@@ -91,6 +95,7 @@ def load_options():
 
   parser.add_option("--adjust", dest="adjustment", type="float", default=None)
   parser.add_option("--reset", dest="reset", action="store_true", default=None)
+  parser.add_option("--light", dest="light_output", type="str", default=None)
```

Comment:
I think this should be action="store_true" and maybe the option can be called "--use-light".

Reply:
I just took `--xrandr` as an example here. But I can make that change if you think this would be better.

Reply:
I feel like '--light' will be confusing (since "light" has many meanings). but also would like to change it to "--use-xrandr" if we go with "--use-light" for consistency's sake. 

perhaps something like "--backend=[xbacklight|light|xrandr]" would make the most sense in the future, but will require more changes. i'm usually up for doing the least amount of work (both on the developer and end-user's part)

so all that said: i'm fine with "--use-light" and "--use-xrandr" and if someone's autolux breaks because they were using xrandr, hopefully they will see the new option quickly enough

Reply:
I would also support doing the least amount of work :)

I have removed the `-ap` options and replaced `--xrandr` and `--light` by boolean options `--use-xrandr` and `--use-light`.

Give it a try and let me know if it works for you (I cannot test with XRandR).